mocks/index.ts

```diff
@@ -2,10 +2,25 @@ export {};
 
 // Only initialize server from a backend context
 if (typeof window === "undefined") {
-  const { server } = require("./server");
+  const startMockServer = async () => {
+    const fs = await import("fs");
+    const path = await import("path");
 
-  server.listen({ onUnhandledRequest: "warn" });
-  console.log("Mock service worker server installed");
-  process.once("SIGINT", () => server.close());
-  process.once("SIGTERM", () => server.close());
+    const mswDataPath = path.join(
+      __dirname,
+      "../../..",
+      "./mocks/msw.local.json"
+    );
+    if (!fs.existsSync(mswDataPath)) {
+      fs.appendFileSync(mswDataPath, "{}");
+    }
```

Comment:
Probably want to do this in the utility module that writes the MSW data

Reply:
Actually, just committing the file manually (with git add) is probably better than including this in code. .gitignore will still ignore any local changes to the file from runtime.

Reply:
git won't ignore local changes if you add the file; at the end of the test run the file stays modified. I think I'll keep the code that creates the file, but move it to the utility module.